.editorconfig

```diff
@@ -2,9 +2,6 @@ root = true
 
 [*]
 charset = utf-8
-end_of_line = lf
-insert_final_newline = true
-trim_trailing_whitespace = true
```

Comment:
@amaanq, why remove these? Are they not sane defaults (unix style)?

Reply:
Forcing one style is incorrect, someone developing a grammar for something Windows-specific might get peeved off